lib/vanagon/component/dsl.rb

```diff
@@ -276,8 +276,25 @@ def url(the_url)
       # Sets the md5 sum to verify the sum of the source
       #
       # @param md5 [String] md5 sum of the source for verification
+      # md5 is our default
```

Comment:
For the sake of YARDoc, I'd move this comment to line 277 or at the end of line 276. It's a nitpick, but the rendered YARDoc will change where it's displayed based on that.